pkg/apis/core/types.go

```diff
@@ -4097,6 +4097,8 @@ const (
 	ResourceQuotaScopeBestEffort ResourceQuotaScope = "BestEffort"
 	// Match all pod objects that do not have best effort quality of service
 	ResourceQuotaScopeNotBestEffort ResourceQuotaScope = "NotBestEffort"
+	// Match all pod objects that have priority class mentioned
+	ResourceQuotaScopePriorityClass ResourceQuotaScope = "PriorityClass"
```

Comment:
update `IsResourceQuotaScopeValidForResource` to say that priority is valid for both pod object count and pod compute quota resources.

Reply:
noting to myself this is done.